components/file_upload/file_upload.jsx

```diff
@@ -705,3 +711,13 @@ export class FileUpload extends PureComponent {
 const wrappedComponent = injectIntl(FileUpload, {forwardRef: true});
 wrappedComponent.displayName = 'injectIntl(FileUpload)';
 export default wrappedComponent;
+
+// Checks if a data transfer contains files not text, folders, etc..
+// Slightly modified from http://stackoverflow.com/questions/6848043/how-do-i-detect-a-file-is-being-dragged-rather-than-a-draggable-element-on-my-pa
+export function isFileTransfer(files) {
+    if (UserAgent.isInternetExplorer() || UserAgent.isEdge()) {
```

Comment:
do we need to check for edge? seems [it is compatible](https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer)

Reply:
Probably not since that check is over 5 years old, but I don't remember enough about it to remove it. I'd rather leave it for now until we end up cleaning up old references to IE11 and Edge throughout the code since we can do a proper test of it then.